swift/WorkflowUI/Sources/ViewRegistry/ViewRegistry.swift

```diff
@@ -25,6 +25,8 @@ import ReactiveSwift
 /// In order for the registry to handle a given screen type, a view factory
 /// must first be registered using `register(screenType:factory:)`, where the
 /// factory is a simple closure that is responsible for instantiating a view.
+///
+/// Deprecated: Return the appropriate `ViewControllerDescription` from `Screen` instead
 public struct ViewRegistry {
```

Comment:
Should this be annotated as deprecated instead of commented as deprecated?

Reply:
This triggers tons of warnings. Especially on all the `Screen` initializers. I decided to just use a comment here, and use the annotations for the `register` call.